opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -413,12 +413,12 @@ namespace Opm
                          WellState& wellstate,
                          ExtraData& extra )
     {
-        std::map<std::string, RestartKey> solution_keys {{"PRESSURE" , UnitSystem::measure::pressure},
-                                                         {"SWAT" , UnitSystem::measure::identity},
-                                                         {"SGAS" , UnitSystem::measure::identity},
-                                                         {"TEMP" , UnitSystem::measure::temperature},
-                                                         {"RS" , UnitSystem::measure::gas_oil_ratio},
-                                                         {"RV" , UnitSystem::measure::oil_gas_ratio},
+        std::map<std::string, RestartKey> solution_keys {{"PRESSURE" , RestartKey(UnitSystem::measure::pressure)},
+                                                         {"SWAT" , RestartKey(UnitSystem::measure::identity)},
+                                                         {"SGAS" , RestartKey(UnitSystem::measure::identity)},
+                                                         {"TEMP" , RestartKey(UnitSystem::measure::temperature)},
+                                                         {"RS" , RestartKey(UnitSystem::measure::gas_oil_ratio)},
+                                                         {"RV" , RestartKey(UnitSystem::measure::oil_gas_ratio)},
                                                          {"SOMAX", {UnitSystem::measure::identity, false}},
```

Comment:
Strange that this line compiles with no change. Do you know why this does not need the explicit construction?